roles/common/tasks/config_validations.yml

```diff
@@ -179,3 +179,10 @@
     - oauth_jwks_uri
   when: oauth_enabled | bool
   tags: validate
+
+- assert:
+    that:
+      - not fips_enabled|bool
+    fail_msg: "OAuth is not supported on FIPS enabled cluster yet. Please disable FIPS or OAuth to setup the cluster."
+  when: oauth_enabled | bool
```

Comment:
are we skipping the <component>_oauth_enabled because those dont have fips?

Reply:
yes, fips support is not present in other components for now

Reply:
Also curious how does this play out in case of colocated scenario. Can fips be deployed in colocated mode. if it can then essentially non broker components would also be on fips despite them not supporting fips.